### PR TITLE
bump ocm sdk version

### PR DESCRIPTION
**Pull request by clcollins**
- Adds Goreleaser release on new tag creation
- Adds Goreleaser release on new tag creation
- Bump ocm-sdk to v0.1.205

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2.3.3
+        with:
+          fetch-depth: 0 # See: https://goreleaser.com/ci/actions/
+
+      - name: Set up Go
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15.2
```
Comment (georgettica) [inline .github/workflows/create_release.yml:26]: This is not the version is go.mod

**Comment by georgettica** [comment]
Tests pass
Please squash and I'll lgtm again and unhold

**Comment by georgettica** [comment]
Good enough for me
/unhold
/lgtm

File: .github/workflows/create_release.yml (hunk 1 above)
Comment (clcollins) [reply .github/workflows/create_release.yml]: Updated to 1.16.8

**Pull request merged**